games-arcade/freegish/freegish-1.0.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="A free version of the game Gish"
+DESCRIPTION="FreeGish is an open-source, freely redistributable version of Gish, a physics-based platforming game featuring a ball of tar."
+HOMEPAGE="http://github.com/freegish/freegish"
+COPYRIGHT="2012-2014 Joris van de Donk"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="git://github.com/freegish/freegish.git#f53d915381c2dd9889dbebdff0541b04c6065067"
+PATCHES="freegish-1.0.patchset"
+
+ARCHITECTURES="x86 x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	freegish$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libsdl$secondaryArchSuffix
+	lib:openal$secondaryArchSuffix
+	lib:zlib$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libvorbis$secondaryArchSuffix
+	devel:libopenal$secondaryArchSuffix
+	devel:libsdl$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:make
+	cmd:gcc
```

Comment:
You probably need to build openal yourself and then remove $secondaryArchSuffix everywhere. You can't link gcc4 libs with gcc2 binaries.